.eslintrc.json

```diff
@@ -13,6 +13,9 @@
             "ignoredNodes": [
                 "ConditionalExpression"
             ]
-        }]
+        }],
+        "func-names": ["error", "as-needed"],
+        "no-alert": 0,
+        "linebreak-style": 0
```

Comment:
What is this for?

Reply:
`"func-names": ["error", "as-needed"],`
- stops eslint complaining about unnamed functions which it can infer the name of.

`"no-alert": 0,`
- stops eslint from complaining about alerts

`"linebreak-style": 0`
- stops eslint from complaining about windows linebreak.
Pushing to github should standardize the linebreak style on github (if it has already been set up).
Checking for linting errors in windows is fairly annoying in any case without this rule.

Reply:
> stops eslint from complaining about alerts

Does this mean we won't get if there are alerts but get to know if there are erroers?

Reply:
I don't really understand what you mean? Alerts are just these pieces of UI which many people find annoying:
![image](https://user-images.githubusercontent.com/16630400/106352583-50765380-631f-11eb-8310-9ab15acd86a3.png)

Reply:
Ah, my bad. I confused "alert" with "warning" in a rush.